app/views/file_attachments/show.html.erb

```diff
@@ -12,11 +12,17 @@
 
     <hr class="govuk-!-margin-bottom-6 govuk-!-margin-top-6"/>
 
-    <dl>
-      <dt class="govuk-body">Title</dt>
-      <dd class="govuk-body"><%= @attachment.title %></dd>
-      <dt class="govuk-body">Attachment link markdown:</dt>
-      <dd class="govuk-body"><%= t("file_attachments.index.attachment_link_markdown", filename: @attachment.filename) %></dd>
-    </dl>
+    <div class="govuk-body">
+      <%= render "govuk_publishing_components/components/attachment_link", {
+        attachment: {
+          title: @attachment.title,
+          url: "#",
+          content_type: @attachment.content_type,
+          file_size: @attachment.byte_size,
+          number_of_pages: @attachment.number_of_pages,
+        },
+        target: "_blank"
+      } %>
+    </div>
```

Comment:
Oh we should still have some the markdown provided for this - just noticed running locally.

Reply:
I've added the attachment link markdown back in 👍